sdk/servicebus/service-bus/src/index.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-/// <reference lib="es2015" />
-/// <reference lib="esnext.asynciterable" />
+// / <reference lib="es2015" />
```

Comment:
Triple-slash references need to start with three slashes (no spaces) and should not be modified. These are no longer working.

Reply:
I think this probably happened because of prettier configs 

> I'm assuming we don't actually want to commit the report.html files?

Yes i don't want to commit that file

Reply:
fixed, i added additional marker to the rule spaced-comment so that it identifies the pattern /// too as a comment

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/3005/files#diff-faf896479b9399a3c04344052079d8d4R38